scripts/issue/330/withdraw_digg_from_tcl_q2.py

```diff
@@ -0,0 +1,38 @@
+from decimal import Decimal
+
+from brownie import interface
+
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import registry
+
+
+def main():
+    """
+    withdraw gdigg from the slp_wbtcdigg tcl position to cover emissions
+    for 2022-q2
+    """
+
+    safe = GreatApeSafe(registry.eth.badger_wallets.treasury_vault_multisig)
+    trops = GreatApeSafe(registry.eth.badger_wallets.treasury_ops_multisig)
+    wbtc = interface.ERC20(registry.eth.treasury_tokens.WBTC)
+    digg = interface.IUFragments(registry.eth.treasury_tokens.DIGG)
+    slp_wbtcdigg = safe.contract(registry.eth.treasury_tokens.slpWbtcDigg)
+
+    safe.init_badger()
+    # https://forum.badger.finance/t/bip-88-emissions-q2-2022/5533/9
+    # 13 weeks in q2
+    total_mantissa = safe.badger.from_gdigg_to_digg(12.5 * 13) * Decimal(1e9)
```

Comment:
i believe here should be `10 * 13` instead as for `bDIGG` it is 100% auto compounded

Reply:
not so much related with this PR, but what will be the new home for the wd WBTC?

still wbtc/badger curve v2 represents a good deal i think, no?

Reply:
that autocompounded digg has to come from somewhere though ;) it doesnt get minted anymore, but is added to the vault by the rewards (geyser) contract when harvesting; eg see https://etherscan.io/address/0x5b60952481eb42b66bdffc3e049025ac5b91c127#tokentxns